docs/src/enterprise/getstarted/quickstart.md

```diff
@@ -7,6 +7,9 @@ description: Quickstart guides for lakeFS Enterprise
 
 Follow these quickstarts to try out lakeFS Enterprise.
 
+!!! warning
+    ⚠️ fluffy will be deprecated in the upcoming versions and all functionality will be migrated into lakeFS-Enterprise
```

Comment:
This is a warning before a warning - we can merge them into the same box with two items